torrent/torrent.go

```diff
@@ -803,7 +804,8 @@ func (ts *TorrentSession) RequestBlock(p *peerState) (error) {
 
 func (ts *TorrentSession) ChoosePiece(p *peerState) (piece int) {
 	n := ts.totalPieces
-	start := rand.Intn(n)
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Doesn't really matter if we use the default seed. It's random enough for this use.